### ISSUE TITLE
Switching audio output mutes audio until client restart on linux

### ISSUE THREAD

**Issue by NotAFile** [yvt/openspades#710]
Occurs with Pulseaudio on Debian

**Comment by feikname** [comment]
You mean by like, connecting a phone?

**Comment by NotAFile** [comment]
yes, plugging in headphones causes it for me. But if that works depends on the device.

**Comment by NotAFile** [comment]
yes, plugging in headphones causes it for me. But if that works depends on the device. It seems some devices show one sink and then do the switching internally while others show multiple devices and then let the OS do the switching. The latter case is the issue.

**Comment by theunknownxy** [comment]
Can you try to write the following in the "~/.alsoftrc" file and tell whether the problem still occurs?
```
[pulse]
allow-moves = true
```
By default OpenAL soft doesn't like being moved around by pulseaudio.

**Comment by godwhoa** [comment]
@theunknownxy That fixed it for me, thanks!
